### ISSUE TITLE
Move default branch from "master" -> "main"

### ISSUE THREAD

**Issue by jsignell** [dask/dask-jobqueue#488]
@jrbourbeau and I are in the process of moving the default branch for this repo from master to main. 

 - [x] Changed in GitHub
 - [x] Merged PR to change branch name in code (xref #487) 
 
### What you'll see
Once the name on github is changed (the first box above is Xed, or this issue closed), when you try to `git pull` you'll get 

```bash
Your configuration specifies to merge with the ref 'refs/heads/master'
from the remote, but no such ref was fetched.
```

### What you need to do

First: **head to your fork and rename the default branch there**
Then:
```bash
git branch -m master main
git fetch origin
git branch -u origin/main main
```

**Comment by lesteve** [comment]
Thanks a lot for working on this! Let me know if there is something you expect me to do, e.g. follow https://github.com/github/renaming#renaming-existing-branches to do the `master` -> `main` renaming via the Github web UI.

**Comment by lesteve** [comment]
I renamed the default branch through the web UI and merged #487. 

cc @dask/dask-jobqueue for more visibility about the `master` -> `main` renaming.

Thanks @jsignell!

**Comment by jsignell** [comment]
Thanks @lesteve!
